lldb/source/Target/RegisterFlags.cpp

```diff
@@ -370,6 +370,16 @@ void FieldEnum::Enumerator::ToXML(Stream &strm) const {
               escaped_name.c_str(), m_value);
 }
 
+void FieldEnum::Enumerator::log(Log *log) const {
```

Comment:
These seem closer to Dump methods that output to a Log instead of a StringStream.  Maybe DumpToLog()?  I wasn't sure what the method would do based on the name.